includes/admin/meta-boxes/class-wcs-meta-box-schedule.php

```diff
@@ -31,53 +31,63 @@ public static function output( $post ) {
 
 	/**
 	 * Save meta box data
+	 *
+	 * @see woocommerce_process_shop_order_meta
+	 *
+	 * @param int $post_or_order_id
+	 * @param WP_Post|WC_Order $post_or_order_object
 	 */
-	public static function save( $post_id, $post ) {
+	public static function save( $post_or_order_id, $post_or_order_object ) {
```

Comment:
I am renaming these params to make it clear that it could be a post or order.

I don't think renaming will have an impact on backwards compat for 3PD that may be calling this public function.